cli/tests/sample-configs/valid/scope_and_when.toml

```diff
@@ -0,0 +1,10 @@
+#:schema ../../../src/config-schema.json
+
+# --when can be used at the top-level. Useful when config is organized over
+# several files.
+--when.commands = ["log"]
+user.name = "John Doe"
```

Comment:
Changing the date format to relative might make more sense as a config you might want only for the "log" command. https://jj-vcs.github.io/jj/v0.32.0/config/#relative-timestamps

Other options that came to mind you could use are `ui.show-cryptographic-signatures` maybe or `ui.pager`.

It obviously doesn't affect the test much, but changing the user name on log feels a bit weird and easy to fix.